docs/source/learn.rst

```diff
@@ -0,0 +1,252 @@
+.. title:: Learn
+
+.. raw:: html
+
+    <h1 class="ui header">Learn Bayesian statistics</h1>
+
+    <h2 class="ui header">...with others!</h2>
```

Comment:
is this all hand-written?

Reply:
yep. we _could_ do this from a configuration file or something, but I think copy/pasting for each new book is fine.